src/mopi/runner/runner.py

```diff
@@ -36,26 +36,26 @@ def __init__(
         if self.experiment.global_dataloader is not None:
             overwrite_dataloaders_(self.pipeline, experiment.global_dataloader)
         overwrite_model_configs_(experiment, self.pipeline)
+        add_split_category_to_datasource_(self.pipeline, experiment)
+
+        self.unloaded_pipeline = deepcopy(self.pipeline)
         append_parent_path_and_id_(self.pipeline, mask=True)
 
         self.run_path = f"{Const.output_runs_path}/{datetime.datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}/"
         self.store = Store(dict(), self.run_path)
-        add_split_category_to_datasource_(self.pipeline, experiment)
```

Comment:
Moved this up so we can have a ``pipeline`` object that already has this modification but not the ``append_parent_path_and_id_`` modification.